websocket/src/websocket.cpp

```diff
@@ -13,6 +13,7 @@
 
 #if defined(__EMSCRIPTEN__)
 #include <emscripten/emscripten.h> // for EM_ASM
+#include "emscripten/websocket.h"
```

Comment:
Should be "<>" since the file isn't in a subdirectory called "emscripten" relative to this current file.

Reply:
Ah, thanks.